sktime/forecasting/arch/_uarch.py

```diff
@@ -0,0 +1,575 @@
+            simulations=self.simulations,
+            rng=self.rng,
+            random_state=self.random_state,
+            reindex=self.reindex,
+        )
+        full_range = pd.RangeIndex(start=start, stop=end + 1)
+        y_pred = pd.Series(
+            ArchResultObject.mean.values[-1],
+            index=full_range,
+            name=str(self._y.name),
+        )
+        y_pred = y_pred.loc[abs_idx.to_pandas()]
+        y_pred.index = self._horizon.to_absolute_index(self.cutoff)
+
+        return y_pred
+
+    def _predict_interval(self, fh, X, coverage):
+        """Compute/return prediction intervals for a forecast.
+
+        State required:
+            Requires state to be "fitted".
+
+        Accesses in self:
+            Fitted model attributes ending in "_"
+            self.cutoff
+
+        Parameters
+        ----------
+        fh : guaranteed to be ForecastingHorizon
+            The forecasting horizon with the steps ahead to to predict.
+        X :  sktime time series object, optional (default=None)
+            Exogeneous time series for the forecast
+        coverage : list of float (guaranteed not None and floats in [0,1] interval)
+           nominal coverage(s) of predictive interval(s)
+
+        Returns
+        -------
+        pred_int : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level coverage fractions for which intervals were computed.
+                    in the same order as in input `coverage`.
+                Third level is string "lower" or "upper", for lower/upper interval end.
+            Row index is fh, with additional (upper) levels equal to instance levels,
+                from y seen in fit, if y_inner_mtype is Panel or Hierarchical.
+                Entries are forecasts of lower/upper interval end,
+                for var in col index, at nominal coverage in second col index,
+                lower/upper depending on third col index, for the row index.
+                Upper/lower interval end forecasts are equivalent to
+                quantile forecasts at alpha = 0.5 - c/2, 0.5 + c/2 for c in coverage.
+        """
+        if fh:
+            self._horizon = fh
+
+        abs_idx = self._horizon.to_absolute_int(self._y.index[0], self.cutoff)
+        start, end = abs_idx[[0, -1]]
+        start = min(start, len(self._y))
+
+        if X is not None:
+            x = {}
+            for col in X.columns:
+                x[str(col)] = np.array(X[col])
+        else:
+            x = None
+
+        ArchResultObject = self._fitted_forecaster.forecast(
+            x=x,
+            horizon=end - start + 1,
+            params=self.params,
+            start=self.start,
+            align=self.align,
+            method=self.method,
+            simulations=self.simulations,
+            rng=self.rng,
+            random_state=self.random_state,
+            reindex=self.reindex,
+        )
```

Comment:
If `method` is not `analytic` (so `simulation` or `bootstrap`), this will have a property `simulations`, which in turn has `values` property among others. If we calculate the quantiles of these, will it give us the desired results for `_predict_quantiles`?

This is what I was thinking, but I'm not at all confident in this so please keep that in mind.

```python
numpy.quantile(forecast_result.simulations.values[-1], list_of_quantiles, axis=0)
```

Ref. https://bashtage.github.io/arch/univariate/forecasting.html
CC: @fkiraly

Reply:
(don't know I'm not familiar with the `arch` package's interface, sorry)

Reply:
@Vasudeva-bit can you please create an issue to track this, and then resolve this conversation? Then we can merge this PR for now, and evaluate later if the probabilistic logic can be improved.

Reply:
Okay, I shall create an issue regarding the same.